_test/test_chains.go

```diff
@@ -36,7 +36,7 @@ var (
 		GasAdjustment:  1.3,
 		GasPrices:      "0.00samoleans",
 		Debug:          true,
-		Timeout:        "10s",
+		Timeout:        "2m",
```

Comment:
Curious, why the large timeout increases?

Reply:
I changed this back to `10s` because the channel handshakes still use it (at least until the next PR) and the tests time out because of that. The `10s` timeout is used in the prior handshake strategy to determine the timeout for each message attempt, not the entire handshake. The `PathProcessor` handles message timeouts with blocks, so I had wired in the timeout as a context timeout for the event processor, and `10s` is not long enough for an entire handshake. To abide by both strategies, there is now an approximated `processorTimeout` of `timeout * 4 * maxRetries` since there are 4 connection handshake messages and each can retry up to maxRetries, if converting from the prior strategy.